File: codewars/Java/Isogram.java
package com.ems.lunchbe;

import java.util.HashSet;
import java.util.Set;

public class Isogram {
    public static boolean isIsogram(String str) {

        if (str.equals("")) {
            return true;
        }

        str = str.toLowerCase();
        char[] arr = str.toCharArray();

        for (int i = 0; i < arr.length; i++) {
            for (int j = 0; j < i; j++) {
                if (arr[i] == arr[j]) {
                    return false;
                }
            }
        }
        return true;
    }
}

class isogramEx1 {
    public static boolean isIsogram(String str) {
        return str.toLowerCase()
                .chars()
                .distinct()
                .count() == str.length();
    }
}

File: codewars/Js/Isogram.js
function isIsogram(str) {

    if (str === "") {
        return true;
    }

    str = str.toLowerCase();
    let arr = str.split('');

    for (let i = 0; i < arr.length; i++) {
        for (let j = i+1; j < arr.length; j++) {
            if (arr[i] === arr[j]) {
                return false;
            }
        }
    }
    return true;

}

console.log(isIsogram("moose"))